test/OAuth2/Storage/AccessTokenTest.php

```diff
@@ -54,4 +54,29 @@ public function testSetAccessToken(AccessTokenInterface $storage)
         $success = $storage->setAccessToken('newtoken', 'client ID', 'SOMEOTHERID', $expires, '');
         $this->assertTrue($success);
     }
+
+    /** @dataProvider provideStorage */
+    public function testRevokeAccessToken(AccessTokenInterface $storage)
+    {
+        if ($storage instanceof NullStorage) {
+            $this->markTestSkipped('Skipped Storage: ' . $storage->getMessage());
+
+            return;
+        }
```

Comment:
Since this isn't part of the interface, it wouldn't be a bad idea to check the `unsetAccessToken` method exists, and skip this test if it doesn't. We would also want a @TODO to remove this for `2.x`